requirements/dev.txt

```diff
@@ -4,6 +4,7 @@ autopep8==1.2.4
 coverage==4.2
 coveralls==1.1
 django-autofixture==0.12.1
+django-extensions==1.7.8
```

Comment:
Umm....I am not sure where are we using `django-extensions`. Just so that I know: could you please tell me what is it used for?